---
title: Comparative genomics across wild isolate populations of selfing <em>Caenorhabditis</em> species
image: variant-calling.png
order: 2
---

Because of the efforts of a number of highly dedicated scientists and citizen volunteers, we have amassed a large collection of <em>C. elegans</em>, <em>C. briggsae</em>, and <em>C. tropicalis</em> wild strains from throughout the world. These strains represent nearly all known isolation locations where we have found these species. Our goals are to use this large strain collection to identify functional variation for a large number of traits and to understand more about the population genetic forces shaping these genomes. As a first step, we are creating robust new reference genomes and <em>de novo</em> gene models.

Next, we are actively deep sequencing all of the strains for each of the three species using Illumina platforms (minimum 50x depth of coverage). These large sequence data sets allow us to not only identify single nucleotide variants but to dig deeper into other classes of genetic variation, including insertion/deletions, transposon insertions, genomic rearrangements, short-tandem repeats, and copy number variants. We hope that these additional classes of variation will contribute to our understanding of how genetic variation underlies phenotypic variation and genome evolution.
