<div class="panel panel-default">
  <div class="panel-heading"><h4>Scripts and Utilities</h4></div>
  <div class="panel-body">
    <a href="https://gist.github.com/danielecook/8d9198f746c9d2a44d19">depth_of_coverage.py</a><small style="float:right"><em><a href="/People/#Daniel Cook">Daniel Cook</a></em></small><br /><small>Python script for calculating depth of coverage and breadth of coverage by chromosome, nuclear, mtDNA, and genome from a given bam file.</small>
    <hr />

<a href="https://gist.github.com/danielecook/cfaa5c359d99bcad3200">generate_masked_ranges.py</a><small style="float:right"><em><a href="/People/#Daniel Cook">Daniel Cook</a></em></small><br />
    <small>Used to produce a bed file of masked ranges within a hard masked ("NNN") fasta file.</small>

    <hr />

       <a href="https://gist.github.com/danielecook/20bd05e461d1e6301a20">het_polarization.py</a><small style="float:right"><em><a href="/People/#Daniel Cook">Daniel Cook</a></em></small><br /><small>Python script for 'polarizing' or re-calling heterogzygous genotype calls as homozygous reference or alternate genotypes based on bayesian probabilities in inbred or largely homozgyous individuals.</small>
    
</div>
</div>

